### ISSUE TITLE
ERROR: default.html.twig template not found for page: /

### ISSUE THREAD

**Issue by cb3inco** [paulmassen/grav-theme-story#6]
I'm new to GRAV and trying this theme and getting the following error:

ERROR: default.html.twig template not found for page: /

Running Grav v1.7.14

**Comment by FrankVan1982** [comment]
Same here.

**Comment by patikepinski** [comment]
same. No one will help i think, he left for another project maybe because this has root problems

**Comment by paulmassen** [comment]
Oh, I forgot about the existence of this theme. Not sure where you heard about it as it is not in the GPM anymore ?

This theme is special as it use a modular page, and thus it do not have a default page (default.html.twig). In any case, this theme needs demo content and an update for compatibility with latest Grav.

I would suggest you try another theme if you are a beginner with Grav but will let you know once it is updated.

**Comment by patikepinski** [comment]
> Oh, I forgot about the existence of this theme. Not sure where you heard about it as it is not in the GPM anymore ?
> 
> This theme is special as it use a modular page, and thus it do not have a default page (default.html.twig). In any case, this theme needs demo content and an update for compatibility with latest Grav.
> 
> I would suggest you try another theme if you are a beginner with Grav but will let you know once it is updated.

What do you mean? The theme is from Boxify and its giving me this error code.

**Comment by paulmassen** [comment]
What is Boxify ?

**Comment by patikepinski** [comment]
> What is Boxify ?

Like i just wrote above its one of the themes that doesn’t work and shows the error stated above..

**Comment by paulmassen** [comment]
Uh, that's not what you wrote.

Anyway, see the first issue : https://github.com/getgrav/grav-theme-boxify/issues/1

**Comment by FrankVan1982** [comment]
Honestly, I'm not using Grav anymore.
I decided to improve my skills, for example by building websites with Vue.js.

**Comment by patikepinski** [comment]
Yeah i'm giving up on Grav because it seems to be a failed project. Wanted it manly to work on IIS windows but it's been days and still errors cant even create an account.
